test/sanity/classes/templateMethod.wtest

```diff
@@ -0,0 +1,31 @@
+class Golondrina {
+	var energia = 100
+
+	method energia() = energia
+
+	method volar(kms) {
+		energia = energia - self.gastoParaVolar(kms) // Invocacion a método que se va a sobreescribir
+	}
+	
+	method gastoParaVolar(kms) = kms
+}
+
+class NoSeCansa inherits Golondrina {
+	override method gastoParaVolar(kms) = 0
+}
+
+describe "template method" {
+
+  test "behavior for swallow does not change" {
+  	const pepita = new Golondrina()
+	  pepita.volar(50)
+	  assert.equals(50, pepita.energia())
+  }
+
+  test "behavior for subclass is overridden by primitive method" {
+    const pepona = new NoSeCansa()
+    pepona.volar(50)
+    assert.equals(100, pepona.energia()) // keeps initial energy
+  }
+
+}
```

Comment:
This is another example of a test file that I don't fully understand what is testing. Not saying we should drop it, but I'm concerned about having to maintain a lot of large tests that might overlap with others.

Reply:
I do believe this test is not overlapping with any other. I'll just add the missing Scenario comment.